app/controllers/application_controller.rb

```diff
@@ -77,7 +77,12 @@ def touch_current_user
   end
 
   def log_image_data_to_datadog
-    images = Array.wrap(params.dig("user", "profile_image") || params["image"])
+    images = Array.wrap(
+      params.dig("user", "profile_image") ||
+      params.dig("podcast", "image") ||
+      params.dig("organization", "profile_image") ||
+      params["image"],
+    )
```

Comment:
I recognize this is pretty hacky. But as soon as I get even just one of these triggered in production, I'm hoping it'll give me the info I need to finally figure this bug out. Once that happens, all of this is going to be ripped right back out.